### DISCUSSION TITLE
Open5GS and ZMQ (gNB + UE) setup on separate machines/hosts

### DISCUSSION THREAD

**Discussion by ahmadhassandebugs** [herlesupreeth/docker_open5gs#341] [Q&A]
Hello,

I have a setup where Open5GS is running on one machine (M1) and ZMQ-based srsRAN gNB and srsUE on another machine (M2). Both machines are on the local network. I followed the readme file for multihost. Further, I changed the `srsue_5g_zmq.yaml` file and replaced the last 6 lines with `network_mode: host`. I also changed the `SRS_UE_IP` in `.env` to the IP address of the host running gNB and UE.

When I start the srsUE container, the UE doesn't get the IP address for some reason and RRC Release is triggered immediately. Below is a screenshot of M2 (left tab is srsGNB and right tab is srsUE). Any kind of help will be appreciated!

![Screenshot from 2024-07-19 17-51-10](https://github.com/user-attachments/assets/5ec713cb-49b8-4031-a033-f1e4e7df911c)

Thanks!

**Comment by herlesupreeth** [comment]
Can you please attach the env files used in the two hosts?

**Comment by ahmadhassandebugs** [comment]
Sure. Here are the environment file:

- [open5gs-host-env](https://github.com/user-attachments/files/16322609/open5gs-host.txt)
- [srsrran-ue-host-env](https://github.com/user-attachments/files/16322610/srsrran-ue-host.txt)

**Reply by herlesupreeth** [reply]
Thanks for the env files. I see that UE is receiving a RRC Release. Can you please send me the F1AP, NGAP and pcap taken on open5gs machine?

**Reply by ahmadhassandebugs** [reply]
Sorry for the delay. I have fixed the issue. I was using OPc instead OP in open5gs userdb config. Thanks!
